src/control/lib/control/auto.go

```diff
@@ -477,6 +477,22 @@ func checkStorage(log logging.Logger, numPmem, reqNumNvme int, storageSet *HostS
 	return pmemPaths, bdevLists, nil
 }
 
+func calcHelpers(log logging.Logger, targets, cores int) int {
+	helpers := cores - targets - 1
```

Comment:
Is there a need to do any validation to the targets and cores to make sure they are positive?

Reply:
Whilst it's probably sensible to future proof the code by adding the check, in both the calls to this function it would be impossible for helpers to end up negative (cores > targets guaranteed in both calls).